.agents/plans/mcp-inspector.md

```
@@ -0,0 +1,317 @@
+
+## Auth modes
+
+### Mode 1: `storageState` (headless, recommended)
+
+User exports their browser session once:
+```bash
+npx playwright open --save-storage=session.json https://your-app.com
+```
+
+Then passes the file path when calling the tool:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)